docs/developer-guide/tutorials/updating-schema.md

```diff
@@ -25,7 +25,7 @@ In order to do that, you'll need to:
 4. Re-generate the `*-Data.js` migration inside `db/migrations`:
     ```bash
     # First, remove the old migration file(s)
-    rm db/migrations/*-Data.js
+    `rm db/migrations/*-Data.js`
```

Comment:
Also I don't think these backticks are needed, the proper way to write this would be `$(rm db/migrations/*-Data.js)` but I don't see the point if the output is not assigned to a variable.